BraintreeDropIn/BTPaymentSelectionViewController.m

```diff
@@ -187,9 +187,16 @@ - (void)configurationLoaded:(__unused BTConfiguration *)configuration error:(NSE
                 }
             }
 
-            // Always add Cards
-            [activePaymentOptions addObject:@(BTUIKPaymentOptionTypeUnknown)];
-            
+            NSArray *supportedCardTypes = [self.configuration.json[@"creditCards"][@"supportedCardTypes"] asArray];
+            for (NSString *supportedCardType in supportedCardTypes) {
+                BTUIKPaymentOptionType paymentOptionType = [BTUIKViewUtil paymentOptionTypeForPaymentInfoType:supportedCardType];
+                if ([BTUIKViewUtil isPaymentOptionTypeACreditCard:paymentOptionType]) {
```

Comment:
The supported card types array returned can include Apple Pay/Android Pay "card" information which is why we need to do a slightly deeper inspection.